index.js

```diff
@@ -16,46 +20,71 @@ var defaultOptions = {
 module.exports = {
   name: 'ember-bootstrap',
 
-  included: function included(app) {
+  included(app) {
```

Comment:
What versions of Node are we supporting? Does the support for Ember 2.3+ also include ember-cli 2.3+? They only dropped Node 0.12 support in ember-cli 2.10, so can we rely on ES6 syntax? I don't think Babel is used for ember-cli, is it? This was pointed out when I made similar changes in the PR against my fork.